package.json

```diff
@@ -69,7 +69,7 @@
 		"build-desktop:source": "run-s build-desktop:config build-desktop:server build-desktop:client build-desktop:static",
 		"build-desktop:config": "node desktop/bin/build-desktop-config.js",
 		"build-desktop:secrets": "node desktop/bin/build-desktop-secrets.js",
-		"build-desktop:server": "BROWSERSLIST_ENV=server CALYPSO_ENV=desktop webpack --config client/webpack.config.desktop.js --display errors-only",
+		"build-desktop:server": "BROWSERSLIST_ENV=server CALYPSO_ENV=desktop webpack --config client/webpack.config.desktop.js --stats-preset errors-only",
```

Comment:
`--display` was selecting a stats preset (https://github.com/webpack/webpack-cli/blob/v3.3.12/bin/cli.js#L146). 

`--stats-preset` is the explicit, equivalent flag.